src/components/network-chart/index.js

```diff
@@ -82,16 +82,11 @@ class NetworkChartImpl extends React.PureComponent<Props> {
   componentDidMount() {
     this.focus();
   }
-  componentDidUpdate(prevProps) {
-    if (
-      this.props.scrollToSelectionGeneration >
-      prevProps.scrollToSelectionGeneration
-    ) {
```

Comment:
Ah @mariagus you went a bit too fast here :) we definitely need the first condition, otherwise we'll call `scrollSelectionIntoView` everytime the component updates, which can be a performance problem.

Reply:
Ah ok! Makes sense :)